test/integration/test_multiple_model_endpoint.py

```diff
@@ -55,19 +55,24 @@ def modulevolume():
 
 @pytest.fixture(scope='module', autouse=True)
 def container(request, docker_base_name, tag):
+    module_dir = os.path.join(resource_path, 'module')
+    model_dir = os.path.join(resource_path, 'models')
+
     test_name = 'sagemaker-sklearn-serving-test'
     try:
         command = (
             'docker run --name {} -p 8080:8080'
-            ' --mount type=volume,source=dynamic_endpoint_model_volume,target=/opt/ml/model,readonly'
-            ' --mount type=volume,source=dynamic_endpoint_module_volume,target=/user_module,readonly'
+            # ' --mount type=volume,source=dynamic_endpoint_model_volume,target=/opt/ml/model,readonly'
+            # ' --mount type=volume,source=dynamic_endpoint_module_volume,target=/user_module,readonly'
+            ' -v {}:/opt/ml/model'
```

Comment:
for my understanding, why were these arguments `module_dir` and `model_dir` added with these changes?